utils/obsproc/applications/gdas_obsprovider2ioda.h

```diff
@@ -26,6 +27,8 @@ namespace gdasapp {
         Rads2Ioda conv2ioda(fullConfig, this->getComm());
         conv2ioda.writeToIoda();
       } else if (provider == "GHRSST") {
+        oops::Log::info() << "GHRSST!" << std::endl;
```

Comment:
look at what's done for the other provider, and remove the log. You clearly have not looked at the output, that code is not writing a ioda file.